ui/src/components/Cards/InlineSourceCard.tsx

```diff
@@ -237,11 +237,7 @@ export const InlineSourceCard: React.FC<{
                           : source?.displayName
                       }
                     >
-                      <Typography
-                        fontWeight={600}
-                        fontSize={12}
-                        sx={{ wordBreak: "keep-all" }}
-                      >
+                      <Typography fontWeight={600} fontSize={12}>
```

Comment:
Seems like this text should be centered?
```suggestion
                      <Typography fontWeight={600} fontSize={12} textAlign="center">
```
![Screen Shot 2023-05-30 at 2 25 51 PM](https://github.com/observIQ/bindplane-op/assets/10042942/a62cbd64-3e5d-45ca-90ff-5560cb18a8a3)
vs
![Screen Shot 2023-05-30 at 2 25 32 PM](https://github.com/observIQ/bindplane-op/assets/10042942/86930375-7a6a-4cf1-8903-ab5951728171)

Reply:
Guess this is q question for Ryan -- I think it's changed back and forth more than once already!

Reply:
Product decision is to leave as is